MC-2093 Document Auth Token Management UI (#256)
hazelcast/management-center-docs: docs/modules/integrate/pages/clustered-rest.adoc

Label: 'test_user_2021-07-07T17:24 EEST'
```

> ==== Using Auth Token Management UI

Reviewer: ```suggestion
==== Using Auth Token Management UI

```

image:ROOT:SettingsToolBar.png[alt=Clustered Rest Provide Headers, align="center"]

. Click *Settings* in the toolbar.
+
image:ROOT:AuthTokenManagementPage.png[alt=Auth Token Management Page, align="center"]

. Navigate to the *API* tab.

> . Select the user that you want to create token for. Only admin can create token for other users.

Reviewer: ```suggestion
. Select the user that you want to create a token for. Only admin can create tokens for other users.
```

> . Enter the label of the token that you want to create. Default value has `username_creationTime` format.

Reviewer: ```suggestion
. Enter the label of the token that you want to create. Default value has the `username_creationTime` format.
```

> . Click *Create Token*

Reviewer: ```suggestion
. Click *Create Token*.
```

You will see the created token on the right side.

image:ROOT:AuthTokenManagementPage_createdToken.png[alt=Created Token, align="center"]

You can copy your token to the clipboard.

> NOTE: The token will not be displayed again so make sure you note it before leaving the page or creating new one.

Reviewer: ```suggestion
NOTE: The token will not be displayed again, so make sure you note it before leaving the page or creating a new one.
```

> You can see the list of created token in table below of the page.

Reviewer: ```suggestion
You can see the list of existing tokens in the table, bottom of the page.
```

image:ROOT:AuthTokenManagementPage_TokenList.png[alt=Created Token List, align="center"]

[cols="20%s,80%a"]
|===
|Field|Description

|Time Issued

> |Issue time of the Token

Reviewer: ```suggestion
|Date and time when the token was created.
```

|User

> |Username of the Token that is created for

Reviewer: ```suggestion
|Username which the token is created for.
```

|Token Label

> |Label of the Token

Reviewer: ```suggestion
|Label of the token.
```

|Action

> |Action for revoking the Token

Reviewer: ```suggestion
|Action for revoking a token.
```
